main.go

```diff
@@ -43,6 +43,16 @@ var (
 	objectIDOID     = append(baseOID, 6)
 	labelOID        = append(baseOID, 9)
 
+	// YubiHSM origins
+	// Do not change the order of these items, as the order relates to a bitmask
+	origins = []string{
+		"generated",
+		"imported",
+		"undefined",
+		"undefined",
```

Comment:
"Undefined" is repeated. Is that intentional?

Reply:
Yes, the labels correspond to bitmasks, and as far as I know, 0x0004 and 0x0008 are undefined.